frame/benchmarking/src/lib.rs

```diff
@@ -1220,6 +1220,11 @@ macro_rules! impl_benchmark_test_suite {
 									},
 									$crate::BenchmarkError::Override(_) => {
 										// This is still considered a success condition.
+										$crate::log::error!("WARNING: benchmark error overrided");
```

Comment:
Would it be helpful to still log `benchmark_name` with the logs in `Override` and `Skip`? Might bake it more clear whats going on

Reply:
Why log with `error!` if it's a warning?

Reply:
@apopiak otherwise it would not appear in the output by default